Readme.architecture.md

```diff
@@ -65,11 +65,11 @@ A good practice may be the syntax `PHONE:[ICAO]:[POS](:[LINE])`, like `PHONE:EDD
 
 Radio station support
 ----------------------------
-This is implemented modular trough a special set of mumble bots. The bots behave as ordinary mumble clients supplementing FGCom-mumble plugin information so the pilots client plugins will behave correctly. From the pilots view, they are just more ordinary clients.
+This is implemented modular trough a special set of mumble bots. The bots behave as ordinary mumble clients, supplementing FGCom-mumble plugin information so the pilots client plugins will behave correctly. From the pilots view, they are just more ordinary clients.
 
 ### ATIS playback
-A `fgcom-radio-playback`-bot can connect to the mumble server. For that to work properly, he will be called with the needed information: frequency to send on and its location. This information will be broadcasted over the mumble plugin interface, so the other mumble pugins of the pilots can pick it up. From then on, the bot behaves as an ordinary radio client from the view of the plugins.
-The bot will read a specified audio file and braodcast it on the selected frequency, until either he is killed or the audio file is deleted (then he kills himself).
+A `fgcom-radio-playback`-bot can connect to the mumble server. For that to work properly, it will be called with the needed information: frequency to transmit on and its location. This information will be broadcast over the mumble plugin interface, so the other mumble pugins of the pilots can pick it up. From then on, the bot behaves as an ordinary radio client from the view of the plugins.
+The bot will read a specified audio file and broadcast it on the selected frequency, until either it is killed or the audio file is deleted (then the bot terminates).
 
```

Comment:
First i was inclined to "hang itself" but that would had indicated a dangling process (kidding)

Honestly, thank you for this, I didn't want to use violent language here but was not able to come up with something better :) :+1: